src/runtime_src/core/common/query_requests.h

```diff
@@ -688,6 +689,16 @@ struct xclbin_uuid : request
   get(const device*) const = 0;
 };
 
+struct dtbo_id : request
+{
+  using result_type = int;
+
+  static const key_type key = key_type::dtbo_id;
+
+  virtual boost::any
+  get(const device*, const boost::any& slot_id) const = 0;
```

Comment:
Let's add a `using slot_id_type = <type> to in`dicate what the type of the `slot_id` argument is.  Then also use that slot_id_type in the implementation when casting the` boost::any`.

I know none of the structs in this file have any significant documentation, but could you add a one liner on 691 explaining briefly what dtbo_id is?